service/src/test/java/uk/gov/mca/beacons/api/accountholder/application/AccountHolderServiceIntegrationTest.java

```diff
@@ -20,6 +25,8 @@
 import uk.gov.mca.beacons.api.accountholder.domain.events.AccountHolderCreated;
 import uk.gov.mca.beacons.api.accountholder.domain.events.AccountHolderUpdated;
 
+@TestPropertySource(locations = "file:main/resources/application.yml")
```

Comment:
Can you not use like test.properties that is checked in, but has $ENV_VAR or "" for each line

Then pull the ENV var out our GH secrets?

I can't image there are that many that need to be private?

Reply:
`@TestPropertySource(locations = "classpath:test.properties")`

Reply:
Now using a test application.yml with secrets coming from exported ENV vars